applications/ShapeOptimizationApplication/tests/algorithm_trust_region_test/parameters.json

```diff
@@ -59,7 +59,7 @@
         },
         "optimization_algorithm" : {
             "name"                    : "trust_region",
-            "max_iterations"          : 3,
+            "max_iterations"          : 4,
```

Comment:
Is there a special reason for this?

Reply:
Yes, I noticed that in the trust region algorithm, there is one mechanic which first is active AFTER  three steps (i.e. the determination of the step length). This step uses the logger, which is why I noticed it.